metricbeat/docs/gettingstarted.asciidoc

```diff
@@ -114,10 +114,22 @@ endif::[]
 
 ifeval::["{release-state}"!="unreleased"]
 
-["source", "shell", subs="attributes"]
-------------------------------------------------
-docker pull {dockerimage}
-------------------------------------------------
+See <<running-on-docker, Running on Docker>> for deploying Docker containers
+
+endif::[]
+
+[[kubernetes]]
+*kubernetes:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {stack-version} of {beatname_uc} has not yet been released.
```

Comment:
perhaps this conditional should be done in the `Running on Kubernetes` page? There you could say this and link to the current released version, cc @dedemorton

Reply:
same for `Running on Docker`

Reply:
I don't think it hurts to have it here for consistency with other platforms (also prevents users from following an extra link), but you should also have the conditional coding where you show any version-specific pull/download commands. Not sure we do that consistently, but we should.

We do this because users end up on the wrong page in the docs. We don't want to show commands that don't work. Note that there's also a prerelease state that might be relevant depending on the specific command you want to show (or not show). You can see the settings for `release-state` in https://github.com/elastic/beats/blob/master/libbeat/docs/version.asciidoc. For example, if you look at that file in the 6.5 branch, you'll see that it's set to `released`. For master, it's set to `prerelease` because we've already released the 7.0 alpha. Make sense?

Reply:
There are no version-specific instructions.  @dedemorton can you make the changes needed please?  I won't be able to work on this until tomorrow, and I don't understand the problem.

Reply:
Looked more closely and it looks like the conditional coding is already in place. I see a few things I can tweak, so I'll do that. I think Carlos is right. I'll take care of his comment.

Reply:
@DanRoscigno Pushed my commits. Let me know if you have any questions/issues.

Reply:
I think I understand now, thanks!!

Reply:
Love this change, thanks @dedemorton !
![image](https://user-images.githubusercontent.com/25182304/49558085-df21ee00-f8d7-11e8-834f-2dbde6350110.png)